### PR TITLE
Empty Total Routing RSI/MCI

### PR DESCRIPTION
**Pull request by dwyeradam**
Zero or empty routing for Total Retail Turnover and Employment questions for schemas:
1_0102, 1_0112, 1_0203, 1_0205, 1_0213 and 1_0215

Completion of https://github.com/ONSdigital/eq-survey-runner/pull/1509 that was merged early.

### CODE REVIEW

**Review by jonnyshaw89 — APPROVED** [review]

File: data/en/1_0112.json [hunk 1]
```diff
@@ -495,6 +497,54 @@
...
+                                    "value": "No",
+                                    "id": "confirm-zero-employees-answer",
+                                    "condition": "equals"
+                                }],
+                                "block": "total-employees"
+                            }
+                        },
+                        {
+                            "goto": {
+                                "block": "changes-in-employees"
```
Comment (jonnyshaw89) [inline data/en/1_0112.json:543]: This should route to `employees-breakdown`

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/1_0112.json (hunk 1 above)
Comment (jonnyshaw89) [reply data/en/1_0112.json:543]: Im wrong. Ignore me

**Review by jonnyshaw89 — APPROVED** [review]

File: tests/functional/spec/features/default_value/default_value.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const helpers = require('../../../helpers');
+
+const QuestionPage = require('../../../pages/features/default_value/number-question.page.js');
+const Summary = require('../../../pages/features/default_value/summary.page.js');
+
+describe('@watch Feature: Default Value', function() {
```
Comment (shahi645) [inline tests/functional/spec/features/default_value/default_value.spec.js]: Remove `@watch`.

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Comment by shahi645** [comment]
We shouldn't merge this until SDX changes are merged in, otherwise they will just get transactions nacking. 
https://github.com/ONSdigital/sdx-transform-cs/pull/147

**Comment by jonnyshaw89** [comment]
@shahi645 not our problem.

**Comment by shahi645** [comment]
yeah but SDC is our service for the sake of not causing them an issue im sure we can hold off merging for like a day

**Comment by jonnyshaw89** [comment]
@shahi645 if we were going down that road then we shouldn't have merged this PR
https://github.com/ONSdigital/eq-survey-runner/pull/1512

If we are going to get these deployed for tomorrow then we need to get them in and tested asap

**Comment by shahi645** [comment]
i completely agree, but we cant test e2e without SDX changes, we can still test locally

**Review by shahi645 — APPROVED** [review]

**Pull request merged**